### PR TITLE
Use default partition for powerVM

### PR DESCRIPTION
**Pull request by tinawang123**
Use default partition for powerVM

- Failed job:  https://openqa.suse.de/tests/13916387#step/installation/3
- Verification run: https://openqa.suse.de/tests/13922380

### CODE REVIEW

**Comment by rakoenig** [comment]
LGTM

**Comment by lemon-suse** [comment]
It seems use default partitions works for there existing partitions before installation, and failed with the profile with configured partitions. It is not clear that why existing partitions before installation. https://openqa.suse.de/tests/13916387#step/bootloader_start/35

For this PR, my suggestion is to create a new profile with this change for default partition.  And we can file a new ticket to investigate to clone from a successful job for profile with partitions set, then update the 
data/yam/autoyast/create_hdd_textmode_powervm.xml. At least we can ensure folder '/' with 'max' size.

File: data/yam/autoyast/create_hdd_textmode_powervm.xml [hunk 1]
```diff
@@ -34,6 +24,8 @@
         <services t="list">
           <service>dhcpv6-client</service>
           <service>ssh</service>
+	      <service>tigervnc</service>
+          <service>tigervnc-https</service>
```
Comment (jknphy) [inline data/yam/autoyast/create_hdd_textmode_powervm.xml:28]: I'm not sure we need this.

**Comment by lemon-suse** [comment]
> > It seems use default partitions works for there existing partitions before installation, and failed with the profile with configured partitions. It is not clear that why existing partitions before installation. https://openqa.suse.de/tests/13916387#step/bootloader_start/35
> > For this PR, my suggestion is to create a new profile with this change for default partition. And we can file a new ticket to investigate to clone from a successful job for profile with partitions set, then update the data/yam/autoyast/create_hdd_textmode_powervm.xml. At least we can ensure folder '/' with 'max' size.
> 
> I don't know where do you got this failed job. My VR can work well. https://openqa.suse.de/tests/13922380#

From your failed job in description, it is before installation.

**Pull request merged**